awx/main/migrations/0101_v370_unifiedjob_canceled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.4 on 2019-11-25 20:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0100_v370_projectupdate_job_tags'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='unifiedjob',
+            name='canceled_on',
+            field=models.DateTimeField(db_index=True, default=None, editable=False, help_text='The date and time the job was canceled.', null=True),
```

Comment:
this won't match the model field's help text anymore

Reply:
oof, sorry, should have made that change while I was making the other. one sec and I will add it here as well.